roles/lib_openshift/src/doc/label

```diff
@@ -0,0 +1,64 @@
+# flake8: noqa
+# pylint: skip-file
+
+DOCUMENTATION = '''
+---
+module: oc_label
+short_description: Create, modify, and idempotently manage openshift labels.
+description:
+  - Modify openshift labels programmatically.
+options:
+  state:
+    description:
+    - State represents whether to create, modify, delete, or list
```

Comment:
> State represents whether to create, modify, delete, or list

I see the choices below are ` ["present", "absent", "list", "add"]` so then:

* `create` maps to `add`
* `delete` maps to `absent`
* `list` maps to `list` 
* `modify` maps to `present`?


Maybe like this?

```yaml
    description:
    - State controls what to do with the module
    - An new label is added with the `create` state
    - An existing label is removed (or verified as removed) with the `absent` state
    - Applied labels may be enumerated with the `list` state
    - A specific labels existence is determined with the `present` state
```

But I'm not sure which of those states accounts for modifying objects